src/AzureIoTHub.Portal/Server/Helpers/DeviceHelper.cs

```diff
@@ -96,6 +96,37 @@ public static void SetDesiredProperty(Twin twin, string propertyName, object val
             twin.Properties.Desired[propertyName] = value;
         }
 
+        /// <summary>
+        /// Set the ClientThumbprint Property as an array of string.
+        /// </summary>
+        /// <param name="twin">Device twin.</param>
+        /// <param name="propertyName">Property to set (ClientThumbprint).</param>
+        /// <param name="value">Property value.</param>
+        public static void SetClientThumbprintProperty(Twin twin, string propertyName, string value)
+        {
+            ArgumentNullException.ThrowIfNull(twin, nameof(twin));
+
+            var camelCasedPropertyName = propertyName.ToCamelCase();
+            twin.Properties.Desired[camelCasedPropertyName] = new List<string>() { value };
+        }
+
+        /// <summary>
+        /// Checks if the specific property exists within the device twin,
+        /// Returns the corresponding value if so, else returns null.
+        /// </summary>
+        /// <param name="twin">Device twin.</param>
+        /// <param name="propertyName">Property name (ClientThumbprint)</param>
+        /// <returns>Corresponding property value, or null if it doesn't exist.</returns>
+        public static string RetrieveClientThumbprintPropertyValue(Twin twin, string propertyName)
+        {
+            ArgumentNullException.ThrowIfNull(twin, nameof(twin));
+
+            var camelCasedPropertyName = propertyName.ToCamelCase();
+
+            return twin.Properties.Desired.Contains(camelCasedPropertyName) ?
+                twin.Properties.Desired[camelCasedPropertyName][0] : null;
+        }
+
```

Comment:
Why did you add this methods ? 
Why not use normal method instead ?

Reply:
I created two new methods here (get/set) because ClientThumbprint is stored as an array in the device twin, whereas other properties simply are strings.

Reply:
For the name (Camel or Pascal case), we should trust in the parameter name given in the method.

For the value (string or array, or object, we may think about serializing the value in Json before setting the value to the twin instead of creating a new method ...